File: Services/Dto/InventoryDto.cs
using System;
namespace Services.Dto
{
    public class InventoryDto
    {
        public int Id { get; set; }
        public int EquipamentId { get; set; }
        public EquipamentDto Equipament { get; set; }
        public decimal Cost { get; set; }
        public decimal Saleprice { get; set; }
        public bool Isnew { get; set; }
        public bool Istested { get; set; }
        public int Quantity { get; set; }
        public int PartnerId { get; set; }
        public PartnerDto Partner { get; set; }
        public int Warranty { get; set; }
        public DateTime Today { get; set; }
        public string Sn { get; set; }
        public string Driver { get; set; }
        public string Manufactorer { get; set; }
        public string Model { get; set; }
        public string Generation { get; set; }
        public string Capacity { get; set; }
        public string Speed { get; set; }
        public string Comment { get; set; }
        public string Historical { get; set; }
        //The real day that record was inserted into database.
        public DateTime Today_ { get; set; }
        //   public DateTime endOfWarranty { get; set; }
        public string ToSeach { get; set; }
    }
}

File: Views/src/app/_shared/components/table-g/dtos/inventory-dto.ts

import { EquipamentDto } from "./equipament-dto";
import { PartnerDto } from "./partner-dto";

export class InventoryDto {

  id:number;
  equipamentId:number;
  equipament:EquipamentDto;
  cost:number;
  saleprice:number;
  isnew:boolean;
  istested:boolean;
  quantity:number;
  partnerId:number;
  partner:PartnerDto;
  warranty:number;
  today:Date;
  sn:string;
  driver:string;
  manufactorer:string;
  model:string;
  generation:string;
  capacity:string;
  speed:string;
  comment:string;
  historical:string;
  today_:Date;
  toseach:string;

}




















//The real day that record was inserted numbero database.



























